modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldDefaultValue.testcase

```diff
@@ -45,12 +45,130 @@ definition {
 
 	}
 
-	@description = "LPS-163716 - Verify updating the default value configuration for the Picklist field affects the object entries"
+	@description = "LPS-181204 - Verify updating the default value configuration for the Picklist field affects the object entries"
 	@priority = 4
 	test CanUpdateObjectEntryByUpdatingConfiguration {
+		property custom.properties = "feature.flag.LPS-163716=true";
 
-		// TODO pending implementation
+		task ("Given a picklist with several items") {
+			Picklist.addPicklistViaAPI(picklistName = "Custom Picklist 181204");
+
+			for (var itemNumber : list "1,2,3,4,5") {
+				Picklist.addPicklistItemViaAPI(
+					itemKey = ${itemNumber},
+					itemName = "Item Test ${itemNumber}",
+					picklistName = "Custom Picklist 181204");
+			}
+		}
+
+		task ("And given a Custom Object with a picklist field that has the default value set via Expression Builder") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 181204",
+				objectName = "CustomObject181204",
+				pluralLabelName = "Custom Objects 181204");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Picklist",
+				fieldLabelName = "Picklist Field",
+				fieldName = "picklistField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject181204",
+				picklistName = "Custom Picklist 181204");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject181204");	
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject181204");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 181204");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.goToFieldsDetails(label = "Picklist Field");
+
+			Click(locator1 = "ObjectAdmin#SIDEBAR_ADVANCED_TAB");
+
+			Check.checkToggleSwitch(
+				key_toggleSwitchLabel = "Use Default Value",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+			Click(
+				key_optionDefaulValue = "Expression Builder",
+				locator1 = "ObjectAdmin#PICKLIST_INPUT_AS_VALUE_OR_EXPRESSION");
+
+			Type(
+				key_text = "Default Value",
+				locator1 = "TextInput#ANY",
+				value1 = "customObject");
+
+			Button.clickSave();
+		}
+
+		task ("And given an object entry with no item selected") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject181204");
+
+			ObjectAdmin.addObjectStringFieldEntryViaUI(entryValue = "Entry Test");
```

Comment:
Can we use the API to add the entry here?

Reply:
Replaced by api